issuance/crl_test.go

```diff
@@ -151,4 +157,91 @@ func TestIssueCRL(t *testing.T) {
 	_, err = issuer.IssueCRL(&defaultProfile, &req)
 	test.AssertError(t, err, "crl issuance with no IDP should fail")
 	test.AssertContains(t, err.Error(), "must contain an issuingDistributionPoint")
+
+	// A CRL with no entries must not have the revokedCertificates field
+	req = defaultRequest
+	req.Entries = []x509.RevocationListEntry{}
+	res, err = issuer.IssueCRL(&defaultProfile, &req)
+	test.AssertNotError(t, err, "parsing crl with no entries")
+	parsedRes, err = x509.ParseRevocationList(res)
+	test.AssertNotError(t, err, "parsing test crl")
+	test.AssertEquals(t, parsedRes.Issuer.CommonName, issuer.Cert.Subject.CommonName)
+	test.AssertDeepEquals(t, parsedRes.Number, big.NewInt(123))
```

Comment:
Don't need these checks in this test case, but do want to assert that x509 also believes it has no revoked certs:
```suggestion
	test.AssertEquals(t, len(parsedRes.RevokedCertificateEntries), 0)
```